manifests/istio-control/istio-discovery/templates/configmap.yaml

```diff
@@ -267,6 +267,36 @@ data:
     networks: {}
   {{- end }}
 
+  Corefile: |-
+
+    global:15054 {
+        errors
+        log
+        proxy . 127.0.0.1:15053 {
+        }
+
+    }
+
+    .:15054 {
+        errors
+        log
+        health :15056 {
+          lameduck 5s
+        }
+
+        kubernetes cluster.local in-addr.arpa ip6.arpa {
+            pods insecure
+            fallthrough in-addr.arpa ip6.arpa
+            ttl 30
+        }
+        prometheus :9153
+
+        forward . /etc/resolv.conf
```

Comment:
Do you know how people set upstream dns servers in k8s with coredns or normal kubedns? Coz we are missing that data

Reply:
I read about it - in istiod we just use the default - I think it's the 'node' DNS resolv.conf. It shouldn't matter - resolv.conf is what would be used by the pod anyways, regardless of how it is set. There are some options in the pod spec - but I don' think they affect our behavior.